marimo/_plugins/ui/_impl/dataframes/dataframe.py

```diff
@@ -109,6 +120,29 @@ def get_dataframe(self, _args: EmptyArgs) -> GetDataFrameResponse:
             row_headers=get_row_headers(self._value),
         )
 
+    def get_column_values(
```

Comment:
Nice, very clean! Gives confidence in the RPC framework.

Reply:
yep, great job here!